src/App.tsx

```diff
@@ -1,14 +1,62 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+const baseTodo: Todo = {
+  id: 0,
+  title: '',
+  completed: false,
+  userId: 0,
+};
+
+const filterTodos = (sourceTodos: Todo[], filterBy: string, currentQuery: string) => {
+  let filteredTodos: Todo[];
+
+  switch (filterBy) {
+    case 'active':
+      filteredTodos = sourceTodos.filter(todo => !todo.completed);
+      break;
+
+    case 'completed':
+      filteredTodos = sourceTodos.filter(todo => todo.completed);
+      break;
```

Comment:
Use Enum instead of the hardcoded values where it's possible

Reply:
> Use Enum instead of the hardcoded values where it's possible

Good point 👍